modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -76,6 +76,49 @@ def pretty(text):
 base_uri = "http://localhost:8081"
 ----
 
+== Manage Schema Registry ACLs
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+You can use ACLs to control access to Schema Registry resources. You can define fine-grained access on a global level, for example, to allow a principal to read all schemas, or on a per-subject basis; for example, to read and write to only the schemas of a specific subject.
```

Comment:
@Feediver1 
"or on a per-subject basis; for example, to read and modify only the schemas of a specific subject'

'write to' is a tad confusing, sounds like writing to a topic.  Users are modifying a schema/subject usually by posting a new version.